docs/content/documentation/systems/Materials/tensor_mechanics/ComputeFiniteBeamStrain.md

```diff
@@ -0,0 +1,12 @@
+#ComputeFiniteBeamStrain
+
+!syntax description /Materials/ComputeFiniteBeamStrain
+
+##Description
+This class computes the rotation matrix that relates the global coordinate system to the current beam local coordinate system for large rotation scenarios in beam elements. Please refer to [C0TimoshenkoBeam](/C0TimoshenkoBeam.md) for details. 
```

Comment:
This description (and the one in the header for this class) makes it sound like it only computes the rotation matrix. It also computes the strain increments (inherited from the base class). Change the description to make that clear.